### PR TITLE
refactor: update guild member

### PR DESCRIPTION
**Pull request by sillyElvy**
## About

This pull request adds missing methods from member to guild member

## Checklist

- [x] I've ran `pre-commit` to format and lint the change(s) made.
- [ ] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- [ ] I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [x] New feature/enhancement
  - [ ] Bugfix

### CODE REVIEW

**Comment by Toricane** [comment]
Have you fetched upstream

**Comment by sillyElvy** [comment]
> Have you fetched upstream

If you're on the app ignore it, it's a bug (if you open it in the browser it is perfectly fine). The pr is at +22/-1 from the diff

**Comment by sillyElvy** [comment]
> > Have you fetched upstream
> 
> If you're on the app ignore it, it's a bug (if you open it in the browser it is perfectly fine). The pr is at +22/-1 from the diff

but ye its from unstable

File: interactions/api/models/gw.py [hunk 1]
```diff
@@ -251,13 +251,20 @@ class GuildMember(ClientSerializerMixin):
...
     premium_since: Optional[datetime] = field(converter=datetime.fromisoformat, default=None)
     deaf: Optional[bool] = field(default=None)
     mute: Optional[bool] = field(default=None)
     pending: Optional[bool] = field(default=None)
 
+    def __str__(self) -> str:
+        return self.name or ""
+
+    @property
+    def avatar(self) -> Optional[str]:
```
Comment (Toricane) [inline interactions/api/models/gw.py:265]: Add docstring

**Review by Toricane — CHANGES_REQUESTED** [review]
Add a docstring

File: interactions/api/models/gw.py (hunk 1 above)
Comment (sillyElvy) [reply interactions/api/models/gw.py:265]: The other method also doesn't have one xD

Comment (Toricane) [reply interactions/api/models/gw.py:265]: Soooo do we add or no

Comment (sillyElvy) [reply interactions/api/models/gw.py:265]: Ig no since it just returns the normal avatar

**Review by Toricane — APPROVED** [review]
LGTM!

**Pull request merged**